### DISCUSSION TITLE
Serve a static file

### DISCUSSION THREAD

**Discussion by elahmo** [dokku/dokku#6397] [Q&A] [answered]
Hey, I am trying to have a simple app where I can serve a PDF file  representing my CV on `myname.example.com`.

This seems to be harder than I thought, I was checking [docs](https://dokku.com/docs/networking/proxies/nginx/#customizing-the-nginx-configuration) and hoped to find something in that section - as there is already an nginx server running when I created an app and assigned it to a domain, but I was unable to set up the configuration to serve a file.

I want to avoid creating a wrapper in python/golang/something else and a separate repo to push, where all I need is a way to serve a single file from a URL.

Is there an easy way to do this?

**Comment by ferblape** [comment, marked as answer]
Have you tried adding `.static` to the root folder of your repo? https://github.com/dokku/heroku-buildpack-nginx?tab=readme-ov-file#dokku

**Reply by elahmo** [reply]
This seems to do the trick. I can also rewrite the nginx config to allow a catchall URL for this file to be served. Thanks!
